mern-ecommerce/controllers/category.js

```diff
@@ -21,7 +21,7 @@ exports.create = (req, res) => {
         error: errorHandler(err),
       });
     }
-    res.json({ data });
+    res.jsoan({ data });
```

Comment:
"Typo: 'jsoan' should be 'json'. It's a common mistake, but it will cause a runtime error. Please correct it to 'res.json({ data });'."